subscriptions/subscriptions.go

```diff
@@ -148,6 +148,14 @@ func (p *Subscriptions) HasTxPermission(
 		if !user.HasRoleFor(org.Address, db.AdminRole) && !user.HasRoleFor(org.Address, db.ManagerRole) {
 			return false, fmt.Errorf("user does not have admin role")
 		}
+	// check CREATE_ACCOUNT
+	case models.TxType_CREATE_ACCOUNT:
+		// check if the user has the admin role for the organization
+		if !user.HasRoleFor(org.Address, db.AdminRole) && !user.HasRoleFor(org.Address, db.ManagerRole) {
+			return false, fmt.Errorf("user does not have admin role")
+		}
```

Comment:
this is the only significant change to review (i spotted this due to a failing test after adding the `default` case)

the rest is harmless linting